src/github.com/matrix-org/dendron/dendron/main.go

```diff
@@ -197,6 +224,19 @@ func main() {
 	mux.Handle("/_matrix/client/api/v1/login", loginFunc)
 	mux.Handle("/_matrix/client/r0/login", loginFunc)
 	mux.Handle("/_matrix/client/versions", versionsFunc)
+
+	if synchrotronURL != nil {
+		synchrotronReverseProxy := proxy.MeasureByPath(
+			proxyMetrics,
+			httputil.NewSingleHostReverseProxy(synchrotronURL).ServeHTTP,
+		)
+		synchrotronFunc := prometheus.InstrumentHandler(
+			"synchrotron", synchrotronReverseProxy,
+		)
```

Comment:
A line explaining what `synchrotronReverseProxy` and `synchrotronFunc` are doing would be nice.